ci-operator/config/openshift/cluster-api-provider-agent/openshift-cluster-api-provider-agent-master.yaml

```diff
@@ -0,0 +1,26 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+promotion:
+  name: "4.10"
+  namespace: edge-infrastructure
```

Comment:
You are not building any images, so you have nothing to promote. Is that intentional?

Reply:
Thanks,
I used the `make new-repo` command and while I did specify that this repo is building an image somehow it's not in the config.
I'll add the image section.